test/testkit/mocks/loggen/log_producer.go

```diff
@@ -50,20 +75,64 @@ func (lp *LogProducer) K8sObject() *appsv1.Deployment {
 			Labels:    labels,
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: ptr.To[int32](1),
+			Replicas: ptr.To(lp.replicas),
 			Selector: &metav1.LabelSelector{MatchLabels: labels},
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels:      labels,
 					Annotations: lp.annotations,
 				},
-				Spec: corev1.PodSpec{
-					Containers: []corev1.Container{
-						{Name: lp.name, Image: "alpine:3.17.2", Command: []string{"/bin/sh", "-c", `while true
+				Spec: podSpec(lp.load),
+			},
+		},
+	}
+}
+
+func podSpec(load Load) corev1.PodSpec {
+	if load == LoadLow {
+		return alpineSpec()
+	}
+	return flogSpec(load)
+}
+
+func alpineSpec() corev1.PodSpec {
+	return corev1.PodSpec{
+		Containers: []corev1.Container{
+			{
+				Name:  DefaultContainerName,
+				Image: "alpine:3.17.2",
+				Command: []string{"/bin/sh", "-c", `while true
 do
 	echo "foo bar"
 	sleep 500
 done`}},
+		},
+	}
+}
+
+func flogSpec(load Load) corev1.PodSpec {
+	var bytePerSecond string
+	if load == LoadMedium {
```

Comment:
do we need loadMedium now ? May be we can introduce it when we need it ?

Reply:
You are right, we don't use it. I will remove it